src/main/java/projectzelda/game/Avatar.java

```diff
@@ -130,7 +130,8 @@ public void move(double diffSeconds)
 
                 // pick up Bones
                 case BONES:
-                    hit(-0.1);
+                    hit(life = 1.05);
```

Comment:
Is this considered good Java? Is there something I'm missing here? Have you read what the hit function does?